app/reset.php

```diff
@@ -25,8 +25,8 @@
 ********************************************************************************/
 session_start();
 require_once '../inc/connect.php';
-require_once '../inc/functions.php';
 require_once '../vendor/swiftmailer/swiftmailer/lib/swift_required.php';
+require_once '../inc/functions.php';
```

Comment:
is there a reason for putting the functions.php require after swift, or it just happened ?